pkg/ipmi/ipmi.go

```diff
@@ -70,7 +71,7 @@ func (i *IPMI) RawSendRecv(msg Msg) ([]byte, error) {
 		case err == syscall.EINTR:
 			continue
 		case err != nil:
-			return nil, fmt.Errorf("ioctlSetReq failed with %v", err)
+			return nil, fmt.Errorf("ioctlSetReq failed with %w", err)
```

Comment:
"ioctlSetReq:%w may be the most common way to write this.

Reply:
updated